woocommerce/woocommerce-ios
# Refund Details: Product & Qty section

Part 1 of #1681 

In this PR the Product & Qty section title displays in the Refund Details screen and shows the product cell, as long as the refund was created as a "Refund by item" type. Note that this PR doesn't include the "Subtotal" cell because the PR is already too big.

For reference, here is the full Refund Details screen:
<img width="410" alt="Screen Shot 2020-01-02 at 2 47 17 PM" src="https://user-images.githubusercontent.com/1062444/71692247-f6e3d400-2d6e-11ea-9152-dbd149d2b955.png">


## To Test
1. On the website, choose an order and create a refund by item. Refund 2 items or more.
2. After creating the refund, go to the app and navigate to the Order Details for the chosen order.
3. The "Refunded" cell should be the new semantic color, "link", which is an active (pink) color. Tap on the "Refunded" cell.
4. The Refund Details screen should appear, with the "Refund #NNN" title and a Product & Qty section. The cell should show an image (if any), the Product Title (product's name), SKU (if any) and a break down of the refunded amount, such as `$total_price ($item_price x qty)`

### Screenshots
| Order Details | Refund Details |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/1062444/71677205-d6545380-2d47-11ea-9f4b-7e74227d0f85.png" width="350" /> | <img src="https://user-images.githubusercontent.com/1062444/71677225-e53b0600-2d47-11ea-9168-eb2ae3a8493e.png" width="350" /> |

Update release notes:
